README.md

```diff
@@ -5,259 +5,77 @@
-
-- [Install locally with Docker](#install-locally-with-docker)
-- [Run with Docker Compose (Label Studio + Nginx + PostgreSQL)](#run-with-docker-compose)
-- [Install locally with pip](#install-locally-with-pip)
-- [Install locally with poetry](#install-locally-with-poetry)
-- [Install locally with Anaconda](#install-locally-with-anaconda)
-- [Install for local development](#install-for-local-development)
-- [Deploy in a cloud instance](#deploy-in-a-cloud-instance)
-
-### Install locally with Docker
-Official Label Studio docker image is [here](https://hub.docker.com/r/heartexlabs/label-studio) and it can be downloaded with `docker pull`. 
-Run Label Studio in a Docker container and access it at `http://localhost:8080`.
-
-
-```bash
-docker pull heartexlabs/label-studio:latest
-docker run -it -p 8080:8080 -v $(pwd)/mydata:/label-studio/data heartexlabs/label-studio:latest
+For the official documentation we refer to the parent label studio [repository](https://github.com/heartexlabs/label-studio).
+
+# Deployment Guide
+The below is a guide on how to deploy the label studio instance including their ml-backends. The deployment is meant for an EC2 (or other VM) and we assume that the infrastructure has already been set up.
+
+The deployment steps are:
+1. Adjust the docker-compose.yml files for label-studio and label-studio-ml-backend to contain the right volumes.
+2. Build and run label-studio docker and label-studio-ml-backend instances
+3. Setup your label studio instance
+
+## 1. Adjusting Docker Compose
+Adjust the docker-compose files to:
+1. Mount the directory with the data to the `app` service in label studio as well as in all ml-backend services.
+The right structure is something like `/home/ubuntu/data:/label-studio/files`. `/label-studio/files` needs to have a directory called `evaluation` containing the pdfs and `test_png` containing all corresponding png files.
+2. Adjust the port for `app` on which your application shall run. If you want to publish your application and make it easily accessable your the default port 80.
+
+## 2. Build and run docker instances
+In order to build and run your docker instances on your EC2 instance, do:
+1. Clone the repositories on your EC2.
+2. Create a data directory and upload all data for your application in that repository.
+3. Create a new tmux session called ml-backend and label-studio in your EC2: `tmux new -s ml-backend` &  `tmux new -s label-studio`.
+4. Attach to the tmux session using `tmux a -t ml-backend` or `tmux a -t label-studio`. Navigate to the respective docker-compose and do `sudo docker compose build` to build all docker images.
+5. Then, inside the respective tmux session do `sudo docker compose up` and your services should be up and running.
+
+Tmux helps you keep processes alive even if you kill your shell instance. A cheatsheet regarding tmux commands can be found [here](https://tmuxcheatsheet.com).
```

Comment:
Mention that we recommend using Tmux at the beginning of this section, and link to the installation page (https://github.com/tmux/tmux/wiki/Installing).
Like this, it can be confusing, you ask the user to execute `tmux` commands before explaining what it is, or that it might be necessary to install something first.

Reply:
tmux is already installed in our docker. So there should not be any need to install it.